index.bs

```diff
@@ -2429,6 +2431,10 @@ invoker.
 		all {{ArrayBuffer}}s for arrays previously returned by
 		{{AudioBuffer/getChannelData()}} on this {{AudioBuffer}}.
 
+		Note: Because {{AudioBuffer}} can only be created via
+		{{BaseAudioContext/createBuffer()}} or via the {{AudioBuffer}} constructor, this
+		cannot throw.
```

Comment:
Not sure what this note is about. Can you elaborate?

Reply:
This is a note to inform readers that it is intentional that their is no text to catch an exception here, because of the reasons outline in https://github.com/WebAudio/web-audio-api/issues/1861#issuecomment-505979518. It's a bit hard to figure out for people just reading this bit of the text, like it was the case for the people that opened the issue in the first place.

Reply:
ACK.